pages/common/terminalizer.md

```diff
@@ -0,0 +1,23 @@
+# terminalizer
+
+> Utility program which records your terminal and generate animated gifs or share a web player.
```

Comment:
Let us avoid using "your". Also, what do you mean by "share a web player" ?

Reply:
I think just calling them "videos" keeps things simple and clear.

Reply:
LGTM